stravalib/client.py

```diff
@@ -177,8 +186,8 @@ def refresh_access_token(self, client_id, client_secret, refresh_token):
 
         Returns
         -------
-        Dictionary containing the access_token, refresh_token
-        and expires_at (number of seconds since Epoch when the provided access token will expire)
+
```

Comment:
```suggestion
Dictionary containing the access_token, refresh_token
        and expires_at (number of seconds since Epoch when the provided access token will expire)
```